### ISSUE TITLE
[BUG]: Burnout 3: Takedown - "Waterfront" and "Kings of the Road" tracks don't load clouds properly during HW mode.

### ISSUE THREAD

**Issue by Gordo10776** [PCSX2/pcsx2#8665]
### Describe the Bug

I think it's only Waterfront and KOTR, but when these are the first tracks to be loaded, the clouds do not appear to be visible. If another track is loaded first that doesn't have this issue (pretty much all Europe and Far East tracks) then Waterfront/KOTR is loaded next, the clouds from the prior track will take over the Waterfront/KOTR sky. With KOTR, the clouds would have a sunset colour, so clouds from another track would be orange. With Waterfront, it would be white. This also happens at native resolution.

1-1: First track loaded on HW: Waterfront (no clouds)
![2](https://user-images.githubusercontent.com/130807641/232175905-8dd24947-40cd-48d0-ab43-9f7bc8dcf9f3.png)

2-1: First track loaded on HW: Kings of the Road (no clouds)
![1](https://user-images.githubusercontent.com/130807641/232175866-c290cf84-d28a-4274-adf3-f05daf325126.png)

3-1: First track loaded on HW: Downtown (unaffected)
![3](https://user-images.githubusercontent.com/130807641/232176059-32716387-ecf4-4a76-8a32-f8c297b54515.png)

3-2: Second track loaded on HW: Downtown > Waterfront (incorrect clouds taken from Downtown)
![5](https://user-images.githubusercontent.com/130807641/232176092-b0a58fdd-a967-4670-8b3a-46e80b667ffc.png)

3-3: Second track loaded on HW: Downtown > Kings of the Road (incorrect clouds taken from Downtown)
![4](https://user-images.githubusercontent.com/130807641/232176113-93c4b62b-d069-4675-a5ac-84ddd908e87d.png)

4-1: Track loaded during SW: Waterfront (correct clouds)
![6](https://user-images.githubusercontent.com/130807641/232176173-987f4e46-6a15-4524-b936-b5fa4b5ee149.png)

4-2: Track loaded during SW: Kings of the Road (correct clouds)
![8](https://user-images.githubusercontent.com/130807641/232176194-b621032c-b815-4844-8509-39a56b16de97.png)

### Reproduction Steps

- On Hardware Mode (HWM), load an event with "Waterfront" or "Kings of the Road" as the track. Clouds will not appear if they're the first track to be loaded.
- If the first track that's loaded is not those two, loading Waterfront/KOTR after it will have the clouds take over from the prior track that doesn't have the issue.
- Only workaround is to switch to Software Mode before loading Waterfront/KOTR, then go back to HWM when the event's fully loaded. Thankfully, restarting the event during HWM will not affect the clouds in these two tracks.

### PCSX2 Revision

v1.7.4382

### Operating System

Windows 10 (64bit)

### CPU

i7-8700

### GPU

RTX 2060

**Comment by JordanTheToaster** [comment]
Cannot reproduce here.

![Burnout 3 - Takedown_SLUS-21050_20230415023321](https://user-images.githubusercontent.com/80843560/232176868-a85cc2bb-75db-4632-a15c-ad9b922ad6f1.jpg)

![Burnout 3 - Takedown_SLUS-21050_20230415023430](https://user-images.githubusercontent.com/80843560/232176872-10032803-68bb-4259-a655-6d29d3fe3a4f.jpg)

**Comment by Gordo10776** [comment]
> 

Sorry, let me rephrase. The first image is the main problem, you got it. The clouds in KOTR is not supposed to look like that. Image 4-2 in OP has the correct clouds, which can only be loaded via Software Mode (then going back to HW after the event has loaded).

The clouds in your above image in KOTR is somewhat borrowed from the previous track that you've loaded prior. Not sure why that's the case.

- If KOTR/Waterfront is the very first track (in the session) loaded during HW, the sky will be void of clouds.
- If KOTR/Waterfront is NOT the very first track to be loaded during HW, it will borrow the cloud textures from the previous track that doesn't have the issue.
